src/tests/ftest/deployment/target_failure.py

```diff
@@ -340,15 +340,14 @@ def test_target_failure_parallel(self):
         :avocado: tags=deployment,target_failure
         :avocado: tags=target_failure_parallel
         """
-        pool_qty = 2
+        self.pool = []
         self.container = []
 
-        # 1. Create 2 pools and a container in each pool. Use the actual size value
-        # rather than percentage because percentage doesn't work well with multiple
-        # pools. Also, this test focuses on the isolation rather than creating the largest
-        # pools possible.
-        self.add_pool_qty(quantity=pool_qty, namespace="/run/pool_size_value/*")
-        for i in range(pool_qty):
+        # 1. Create 2 pools and a container in each pool. Each pool uses about 40% of the
+        # available storage.
+        self.pool.append(self.get_pool(namespace="/run/pool_size_ratio_40/*"))
+        self.pool.append(self.get_pool(namespace="/run/pool_size_ratio_66/*"))
+        for i in range(2):
```

Comment:
Ah, I was waiting for this size % complication to surface :) It becomes really fun with even more pools...